api/test/fixtures/api.ts

```diff
@@ -63,7 +63,8 @@ export const mockedApi = <T extends Displayable & Identified>(
     },
     {
       pipelineDate: "2222-22-22",
-      contentsIndex: testIndex,
+      articlesIndex: testIndex,
+      eventsIndex: testIndex,
```

Comment:
To prevent a future headache might be worth splitting out a `testArticlesIndex` and a `testEventsIndex` here